src/main.ts

```diff
@@ -358,7 +361,7 @@ export class Purchases {
       Logger.debugLog(
         "Initializing Purchases SDK with Web billing sandbox API Key",
       );
-    } else if (isRCTestStoreApiKey(apiKey)) {
+    } else if (isSimulatedStoreApiKey(apiKey)) {
       Logger.debugLog("Initializing Purchases SDK with RC Test store API Key.");
```

Comment:
```suggestion
      Logger.debugLog("Initializing Purchases SDK with RC Simulated store API Key.");
```

Reply:
Actually, we're planning on leaving anything user facing, like logs with the `Test store` name. `Simulated store` will be the internal name for this store. This is to avoid ambiguity within our codebase, which is tricky with the "test" naming.